website/pages/docs/glossary/what-is-infrastructure-data-lake.mdx

```diff
@@ -0,0 +1,69 @@
+
+## Monitoring
+
+Ingestion, storage, transformation and visualization are the key steps in the pipeline which can bring you pretty far. Though as you move forward you can productionize even more with other tools in the modern-data-stack ecosystem such as monitoring and governance tools.
+
+# Use Cases
+
+The infrastructure data lake is a key component to be able to build a number of popular use-cases in a customizable and scalable way.
+
+## Cloud Asset Inventory
+
+Once all the configuration data is in your data lake it’s easy to create any view or correlation with standard data tools. For example by normalizing resources in different clouds you can create a cross cloud asset inventory view and visualize it. Take a look at how to build an [open source cloud asset inventory with CloudQuery and Grafana](https://www.cloudquery.io/blog/open-source-cloud-asset-inventory-with-cloudquery-and-grafana), which includes pre-built views and dashboards.
+
+## CSPM
+
+Cloud Security Posture Management (CSPM) is a popular solution for security professionals to audit and monitor their cloud configurations for security best practices and their internal policies. Those solutions usually come as pre-packaged dashboards with pre-built rules and custom query languages. The upside of those solutions is that it is easy to get started but often when some customization is needed it’s hard to access the raw data with standard data tools and integrate to any other standard visualization tools which.
+
+## FinOps
+
+Optimizing cost, placing guard-rails and tagging best practices is a  full-time job especially in big cloud environments. But to understand what you need to optimize you need access to both your cost data (such as AWS CUR data) and your infrastructure in queryable manager to be able to drive insights.
+
+For example check out:
+
+- “[Correlating Costs to resources for Azure cost optimization](https://www.cloudquery.io/blog/azure-cost-optimization-with-cloudquery)”
+- “[How to Enrich AWS Cost and Usage Data with Cloud infrastructure Data in Athena](https://www.cloudquery.io/how-to-guides/query-aws-cost-and-usage-athena)”
+
+## CIEM
+
+Cloud Infrastructure Entitlement Management (CIEM) - Is the process of managing identities and privileges in cloud environments. In layman terms is the process of understanding who has access to what. This can be done by querying tables such as [aws_iam_*](https://www.cloudquery.io/docs/plugins/sources/aws/tables) and correlating with other resources or even normalizing users across cloud providers such as [gcp_iam_*](https://www.cloudquery.io/docs/plugins/sources/gcp/tables) . Or even ad-hoc analysis such as “[Finding Cross-Account AWS Event Bridge Usage](https://www.cloudquery.io/blog/how-to-find-cross-account-aws-eventbridge-usage)”
```

Comment:
```suggestion
Cloud Infrastructure Entitlement Management (CIEM) - Is the process of managing identities and privileges in cloud environments. In layman terms is the process of understanding who has access to what. This can be done by querying tables such as [aws_iam_*](/docs/plugins/sources/aws/tables) and correlating with other resources or even normalizing users across cloud providers such as [gcp_iam_*](/docs/plugins/sources/gcp/tables) . Or even ad-hoc analysis such as “[Finding Cross-Account AWS Event Bridge Usage](/blog/how-to-find-cross-account-aws-eventbridge-usage)”
```